### PR TITLE
Add argutils

### PR DESCRIPTION
**Pull request by druvus**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by druvus** [comment]
I will add a PR to remove it from Bioconda once it is merged here.

**Comment by druvus** [comment]
@conda-forge/help-python Could someone please review?

File: recipes/argutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pyyaml
+    - six
+
+test:
+  imports:
+    - argutils
+
+about:
+  home: "https://github.com/eclarke/argutils"
+  license: "GPL-2.0"
```
Comment (isuruf) [inline recipes/argutils/meta.yaml]: `GPL-2.0-or-later`

File: recipes/argutils/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (isuruf) [inline recipes/argutils/LICENSE.txt]: Is there an upstream issue about the missing license file?

Comment (druvus) [reply recipes/argutils/LICENSE.txt]: The license file unfortunately missing in the upstream repo. The license is only stated in the setup.py file.

Comment (druvus) [reply recipes/argutils/LICENSE.txt]: Filed an upstream issue about the missing license file: eclarke/argutils/issues/2

**Pull request merged**